baselines/compute/src/v1/addresses_client_config.json.baseline

```diff
@@ -24,11 +24,11 @@
           "retry_codes_name": "non_idempotent",
           "retry_params_name": "default"
         },
-        "Delete": {
+        "UnaryDelete": {
```

Comment:
I believe we must keep the original method names in this config because this is only a client change and gax will keep using original names.